packages/functional-tests/src/tests/look-at-test.ts

```diff
@@ -7,13 +7,18 @@ import * as MRE from '@microsoft/mixed-reality-extension-sdk';
 
 import { Test } from '../test';
 
-const circleKeyframes = [
-	{ time: 0, value: { transform: { local: { position: { x: 0, y: 0, z: -0.5 } } } } },
-	{ time: 1, value: { transform: { local: { position: { x: 1, y: 1, z: -0.5 } } } } },
-	{ time: 2, value: { transform: { local: { position: { x: 0, y: 2, z: -0.5 } } } } },
-	{ time: 3, value: { transform: { local: { position: { x: -1, y: 1, z: -0.5 } } } } },
-	{ time: 4, value: { transform: { local: { position: { x: 0, y: 0, z: -0.5 } } } } }
-] as MRE.AnimationKeyframe[];
+const circleAnimData: MRE.AnimationDataLike = {
+	tracks: [{
+		target: MRE.ActorPath("target").transform.local.position,
+		keyframes: [
+			{ time: 0, value: { x: 0, y: 0, z: -0.5 } },
+			{ time: 1, value: { x: 1, y: 1, z: -0.5 } },
+			{ time: 2, value: { x: 0, y: 2, z: -0.5 } },
+			{ time: 3, value: { x: -1, y: 1, z: -0.5 } },
+			{ time: 4, value: { x: 0, y: 0, z: -0.5 } }
+		]
+	} as MRE.Track<MRE.Vector3>]
+};
 
 export default class LookAtTest extends Test {

```

Comment:
The monkey head is embedded in the floor for me:
![image](https://user-images.githubusercontent.com/12176807/79381771-55e0e980-7f17-11ea-8c63-1904a65b0f25.png)

Reply:
This one was simple, the `isPlaying` field wasn't set. There are other tests that are more broken though. Investigating.

Reply:
Okay, all the tests are fixed. Some of the broken tests actually exposed bugs that I fixed (i.e. setting the transform of an actor with a rigidbody was broken), so that worked out!